catroid/src/main/java/org/catrobat/catroid/stage/PreStageActivity.java

```diff
@@ -102,6 +102,14 @@ public void onCreate(Bundle savedInstanceState) {
 
 		SensorHandler sensorHandler = SensorHandler.getInstance(getApplicationContext());
 
+		if ((requiredResources & Brick.DATE) > 0) {
```

Comment:
this is not needed, because the calendar is always available and does not need to get initialized. The resources are for things, were we need the user to enable them (like gps data, camera etc.)